### DISCUSSION TITLE
Arabic characters are not displayed in PDF

### DISCUSSION THREAD

**Discussion by fastender** [quarto-dev/quarto-cli#4353] [Q&A] [answered]
Unfortunately, I can't find a way to generate Arabic letters or words via PDF. What could I do, do you have any help for me?

**Comment by rgaiacs** [comment]
Dear @fastender, could you provide a minimal example?

**Reply by fastender** [reply]
Thank you for the prompt report. I best attach a screenshot of what happened. When I render the QMD file, the Arabic texts are not displayed in the PDF.
<img width="1199" alt="Bildschirm­foto 2023-02-15 um 09 30 54" src="https://user-images.githubusercontent.com/22562304/218974409-36cf2341-3df5-49fd-808b-b56d38a525c4.png">

**Reply by rgaiacs** [reply]
I saw answers in Stackoverflow, like https://tex.stackexchange.com/a/12369/22451, mentioning the use of

```
\usepackage{polyglossia}
```

Try add this package as part of `include-before-body` in your YAML header. More at https://quarto.org/docs/reference/formats/pdf.html#includes.

**Comment by fastender** [comment]
Arabic characters are generated separately

Thank you very much for your advice. I have made some progress, but now I have the problem that every single character is displayed separately, so instead of "ذهب جون إلى المنزل" unfortunately "لزنملاىلإنوج" is displayed, what am I doing wrong?

<img width="434" alt="Bildschirm­foto 2023-02-21 um 12 55 01" src="https://user-images.githubusercontent.com/22562304/220338229-ca1431f8-8b18-4217-85b2-788c1b147885.png">


<img width="723" alt="Bildschirm­foto 2023-02-21 um 12 55 59" src="https://user-images.githubusercontent.com/22562304/220338406-125b4069-9edf-4ee2-9a47-2cea4551e8ee.png">

**Reply by baabadkhalid** [reply]
Have you found a way to write arabic in quarto.!?

**Reply by adamiturabi** [reply, marked as answer]
I have a MWE here that may help get started: https://adamiturabi.github.io/quarto-arabic-mwe/
